src/configs/envs.ts

```diff
@@ -116,3 +116,6 @@ export const EXPRESS_REDIS_SENTINEL_CONFIG = JSON.parse(process.env.EXPRESS_REDI
 
 export const EXPRESS_RESPONSE_HEADERS = JSON.parse(process.env.EXPRESS_RESPONSE_HEADERS || '{}');
 export type EXPRESS_RESPONSE_HEADERS = typeof EXPRESS_RESPONSE_HEADERS;
+
+export const { NODE_ENV } = process.env;
+export type NODE_ENV = typeof NODE_ENV;
```

Comment:
We actually never use types, I think we should remove their exports. Also I think `typeof IMPORTED_VAR` is too trivial to abstract

Reply:
I will add an issue to refactor the envs file.